backend/src/controllers/todo.spec.ts

```diff
@@ -5,11 +5,24 @@ import { DirtyTodo, TestTodo } from '../models/todo';
 
 describe('Todos', () =>
 {
-	beforeAll(async() => await mongoose.connect(process.env.DB_URL));
+	let loginToken : string;
```

Comment:
Use token : Token;

Reply:
can't be renamed, I changed into receivedToken because down there on line I extract token from loginResponse.body